strategy/strategy.go

```diff
@@ -24,18 +24,28 @@ func (a *Action) Canonicalize() {
 	}
 }
 
-func (a *Action) IsWithinLimits(min, max int64) bool {
-	return a.Count >= min && a.Count <= max
+func (a *Action) IsWithinLimits(min, max int64) (bool, error) {
```

Comment:
it may be cleaner to just have `IsWithinLimits` and `CapCount` be no-ops when `count == nil`, since the only error case is nil. it will eliminate the error handling, at least.

Reply:
I thought of this, but I couldn't find a good no-op for this method. Returning either `true` or `false` are both misleading and could cause problems.

This is not the case for the current only place that I call this, since I check for a `nil` value before calling `IsWithingLimits`, but I didn't want to tie the method's signature with how it will be used. 

What I can do is to just ignore the error in the agent since, as I mentioned, I will be checking for `nil` beforehand.